.github/workflows/lint_code.yml

```
@@ -14,7 +14,7 @@ jobs:
       with:
         python-version: 3.8
     - name: Install dependencies
-      run: pip install flake8
+      run: pip install flake8<6.0.0
```

I think the `<` character is being interpreted by the shell as a redirection, does this escape work?
```suggestion
      run: pip install 'flake8<6.0.0'
```